### ISSUE TITLE
Outdated

### ISSUE THREAD

**Issue by Hok-2323** [DV8FromTheWorld/JDA#1854]
## General Troubleshooting

- [*] I have checked for similar issues.
- [*] I have updated to the [latest JDA version][download].
- [*] I have checked the branches or the maintainers' PRs for upcoming bug fixes.

## Bug Report

Interaction wiki isn't update to date 

### Expected Behavior

SelectionMenu already came out

### Code Example or Reproduction Steps

N/A

### Code for JDABuilder or DefaultShardManagerBuilder Used

N/A

### Exception or Error

N/A

**Comment by Hok-2323** [comment]
https://github.com/DV8FromTheWorld/JDA/wiki/Interactions

**Comment by Yashar256** [comment]
In regards to the `good first issue` tag, it seems like only collaborators can edit the wiki entries (although I may be mistaken, I haven't used wikis too much).

**Comment by DV8FromTheWorld** [comment]
Yeah, we are working on finding a way to expose the wiki files inside of the standard git repository so that users can create PRs for the Wiki.

Until then, unfortunately, it requires submitting a patch file or a hastebin link. Very annoying, but that's a github restriction.
